src/pages/where-we-work.tsx

```diff
@@ -0,0 +1,64 @@
+import { graphql } from 'gatsby'
+import React, { FunctionComponent } from 'react'
+import SimpleLayout from '../layouts/Simple'
+
+interface Props {
+  data: {
+    allContentfulDataGeoRegion
+    allContentfulDataGeoRegionSubRegion
+  }
+}
+
+const WhereWeWorkPage: FunctionComponent<Props> = ({ data }) => {
+  const pageContext = {
+    pageTitle: 'TODO',
+    siteTitle: 'TODO',
+  }
+
+  return (
+    <SimpleLayout pageContext={pageContext}>
+      {/* page header */}
+      <header>
+        <h1>Where We Work</h1>
+      </header>
+
+      <ul>
+        {data.allContentfulDataGeoRegion.nodes.map((node) => {
+          return (
+            <li>
+              <a href={`./where-we-work/${node.slug}`}>{node.name}</a>
```

Comment:
Suggestion (non-blocking): use Gatsby's `Link` component to avoid a full page reload. Docs:
https://www.gatsbyjs.com/docs/reference/built-in-components/gatsby-link/